model/src/main/proto/spotlight.proto

```diff
@@ -10,29 +10,26 @@ option java_multiple_files = true;
 message Spotlight {
   // Determines the UI element being spotlit.
   oneof feature {
-    // Corresponds to the onboarding screen's next button.
-    SpotlightViewState onboarding_next_button = 1;
-
     // Corresponds to the topic fragment's lessons tab.
-    SpotlightViewState topic_lesson_tab = 2;
+    SpotlightViewState topic_lesson_tab = 1;
```

Comment:
Keep existing indexes and mark the old onboarding one as reserved with a comment.

Reply:
Done - Reserved in spotlightstatedatabase proto. The one mentioned here is a oneOf, from where it was removed. PTAL

Reply:
I have checked this and this change does look good.

Reply:
I think part of it is missing, left one follow-up.